spec/integration/fixtures/no-parameter-section.json

```diff
@@ -0,0 +1,13 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "A simple CFN stack to test KumoKeisei",
+    "Resources": {
+        "KumoTestSecurityGroup": {
```

Comment:
Could this be changed to a different type of object which doesn't require the vpc-id?
A tag perhaps instead?